src/components/modals/loginModal.js

```diff
@@ -47,8 +79,8 @@ export const LoginModal = ({
               <p className="text-sm my-5">You do not have account yet?</p>
               <label
                 onClick={() => {
-                  setShowModalLogin(!showModalLogin);
-                  setShowModalSignUp(!showModalSignUp);
+                  setShowSignInModal(!showSignInModal);
+                  setShowSignUpModal(!showSignUpModal);
```

Comment:
for both cases, avoid using this approach. Here is no problem, because both of them are not dependant on each other, but if so, you may cause infinite rerender loop.
Always use `prevState` if state setter changing itself state
`setShowSignInModal(prevState => !prevState)`